<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>764</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>GlobOpt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>301</width>
      <height>441</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>299</width>
       <height>439</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="function">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>781</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>((x1-1)*(x1-1)+(x2-1)*(x2-1))</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>2</y>
      <width>173</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Введите функцию</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>58</y>
      <width>292</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Введите размерность функции</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>122</y>
      <width>222</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Диапазон вычислений:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>620</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>86</y>
      <width>238</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>321</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Метод поиска</string>
       </property>
       <widget class="QRadioButton" name="scan">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>28</y>
          <width>319</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Метод последовательного сканирования</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="strong">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>319</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Алгоритм глобального поиска Стронгина</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="loma">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>54</y>
          <width>319</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Метод ломаных (метод Пиявского)</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Преобразования</string>
       </property>
       <widget class="QRadioButton" name="mon2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>54</y>
          <width>319</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Монотонное преобразование 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="mon1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>28</y>
          <width>319</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Монотонное преобразование 1</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Без преобразований</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Параметры метода поиска</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Точность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="eps">
            <property name="decimals">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Критерий Липшеца</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="lipsh"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Результаты</string>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>512</y>
          <width>301</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Результаты</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>23</y>
          <width>301</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Количество испытаний</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kolvo"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>301</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Лучшая точка             </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="best"/>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="min">
        <property name="geometry">
         <rect>
          <x>149</x>
          <y>90</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Глобальный минимум</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
